src/nprintml/cli.py

```diff
@@ -15,6 +15,7 @@
 # ensure default steps of pipeline auto-load
 import nprintml.net.step
 import nprintml.label.step
+import nprintml.learn.step
```

Comment:
(FYI: I intend to make the pipeline a tad more automagical to relieve this module of this sort of boilerplate; but, there's little reason to worry about that now.)